package.json

```diff
@@ -101,6 +101,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.25",
     "@fortawesome/free-solid-svg-icons": "^5.11.2",
     "@fortawesome/react-fontawesome": "^0.1.7",
+    "messageformat": "^1.0.0",
```

Comment:
is there a specific reason to use v1.0 rather than the latest? (v2.3)

Reply:
2.3 requires babel 7 (I tried to upgrade this repo to use babel 7 and hit a number of road blocks). I get this error:  ```ReferenceError: Unknown plugin "@babel/plugin-proposal-class-properties" specified in "/home/bconnor11/ml-activities/node_modules/messageformat/package.json" at 0, attempted to resolve relative to "/home/bconnor11/ml-activities/node_modules/messageformat"```

I can use v1.1 though so I'll upgrade to that.